lib/libutee/tee_api_operations.c

```diff
@@ -816,6 +811,16 @@ TEE_Result __GP11_TEE_SetOperationKey2(TEE_OperationHandle operation,
 	return set_operation_key2(operation, key1, key2);
 }
 
+TEE_Result TEE_SetOperationKey2(TEE_OperationHandle operation,
+				TEE_ObjectHandle key1, TEE_ObjectHandle key2)
+{
+	if (operation != TEE_HANDLE_NULL && key1 && key1 == key2)
```

Comment:
TEE_SetOperationKey2() should also panic if operation isn't a valid handle.

Reply:
It will, it is taken care of in `__GP11_TEE_SetOperationKey2()`